src/NuGet.Server/Web.config

```diff
@@ -69,6 +69,8 @@
     on a fixed 1-hour interval.
     -->
     <add key="enableFileSystemMonitoring" value="true" />
+    <!--Set allowRemoteCacheManagement to true to enable the "clear cache" and other cache operations initiated via requests originating from remote hosts.-->
+    <add key="allowRemoteCacheManagement" value="true" />
```

Comment:
I would set default to `false`, to match the default in code and existing behavior.